### ISSUE TITLE
App Crashes on IMMEDIATE Update

### ISSUE THREAD

**Issue by shreyansh-jain** [SudoPlz/sp-react-native-in-app-updates#31]
I'm using version: 1.1.1, and updating the app with the type 'Immediate'. 

The first flow happens smoothly where it asks the user to update the app, but it keeps on installing until it crashes. When I open the app again it shows as updated.

Here is the update call - 
`let update_type = {
    updateType: IAUUpdateKind.IMMEDIATE
  };
  try {
    let response = await this.inAppUpdates.startUpdate(
      update_type
    )
  }
 catch(e) { console.log(e); }`

**Comment by SudoPlz** [comment]
Can you please share a stacktrace at least?

Thanks

**Comment by shreyansh-jain** [comment]
Thanks, Can you please help me generate a stack trace. Since this can only be tested using internal app sharing, I am unable to generate a stack trace.

**Comment by maslenkov** [comment]
> Thanks, Can you please help me generate a stack trace. Since this can only be tested using internal app sharing, I am unable to generate a stack trace.

use `react-native log-android` (https://stackoverflow.com/a/38056698/1514072) from your local console to review logs

**Comment by shreyansh-jain** [comment]
Thanks @maslenkov

**Comment by parth-koshta** [comment]
Hey @shreyansh-jain 
Were you able to fix this? My app crashes after update is installed.

**Comment by shreyansh-jain** [comment]
Nope. Didn't found the fix. But the same package is working perfectly with my other app so I guess the issue is with some other files which are already present in the app.
